doc/integrator/intranet.rst

```diff
@@ -69,7 +70,19 @@ This page lists the changes that must be applied to add such a functionality.
                                    name='user', reify=True
        )
 
-2. In the ``vars`` section of ``vars.yaml`` add
+2. In ``apache/wsgi.conf.mako`` add to ``<Location /${instanceid}/wsgi>``:
+
+   .. code:: apache
+
+       SetEnvIf REMOTE_ADDR ${intranet_ip} intranet=1
+
+   or, depending on the proxies setup:
+
+   .. code:: apache
+
+       SetEnvIf x-forwarded-for ${intranet_ip} intranet=1
```

Comment:
The 2.3 doc is on docker than we don't have any apache folder ...

Reply:
I am not sure I understand. We still have an apache/wsgi.conf.mako (in ./geoportal/c2cgeoportal_geoportal/scaffolds/nondockercreate), so if somebody uses this, this change to the configuration is still necessary?

Reply:
Yes but I think the preferable to have the documentation for the Docker version or specify that's only fro the non Docker version :-)

Reply:
OK